library/ortools/BUILD

```diff
@@ -1,20 +1 @@
-config_setting(
```

Comment:
OS detection utilities were located in `library.ortools`. They are more generic than that, and should be relocated to `util.platform`.